min_z: 0.05
max_z: 10.0
r_search: 0.05
leaf_size: 0.025
downsample: 4
